### DISCUSSION TITLE
Sections in Management and Operations chapter

### DISCUSSION THREAD

**Discussion by becarpenter** [becarpenter/book6#56] [Ideas]
I've added some proposed sections at https://github.com/becarpenter/book6/blob/main/6.%20Management%20and%20Operations/6.%20Management%20and%20Operations.md

Comments, suggestions and above all **new text** are needed.

**Comment by ehorley** [comment]
I have some content I am considering adding. Do you want the focus of this chapter to be around operational differences and how to make decisions for a design or are you assuming those decisions have been made in the architecture and design and the focus is on operations post-implementation?

**Comment by becarpenter** [comment]
Good question. Of course the design and how you operate interact. If you can create your content as a stand-alone new section we can maybe decide where it best fits? (Although I have to do a bit of magic behind the scenes, renaming a section or moving it to another chapter is completely possible. If it happens often, I might write another Python utility to do the necessary edits automatically.)

**Comment by ehorley** [comment]
Okay, do you just want me to do a branch with a new section for the chapter and put my content in that? Then you can merge and then just edit out of main? The alternative is to do a new scratch space chapter/section for blocks of text with a title and we can determine in a sprint where the content belongs. I am happy to do either.

**Comment by becarpenter** [comment]
Go ahead as a new section in that chapter for now. A scratch section for new material is not a bad idea but I need to think if it has any impact on the existing tooling. If I don't see any gotchas, I could just create it as Chapter 98 with a suitable warning to readers.

**Comment by becarpenter** [comment]
Ping to Ed: any PR imminent?
